src/index.html

```diff
@@ -3,12 +3,63 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo_header">
+      <a class="home_link" href="home">
+        <img class="logo_image" src="./images/logo.png"
+        alt="moyoLogo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li>
+            <a class="nav_link is-active" href="Apple"
+            >APPLE</a>
+          </li>
```

Comment:
1. add class name to each item
2. set a correct reference at href
3. apply code styling recommendations
4. use text at lowercase in html and style at css
```suggestion
          <li class="nav_item">
            <a class="nav_link is-active" href="#apple">apple</a>
          </li>
```